blocks/header/header.js

```diff
@@ -93,21 +105,35 @@ function toggleNavSubSection(event) {
   }
 }
 
+function toggleBreadCrumb() {
+  document.querySelector('.menu-option-sublist.l3-nav-menu-options').classList.toggle('active');
+  document.querySelector('.l3-nav-mobile-menu-body').classList.toggle('active');
+  document.querySelector('.l3-nav-mobile-menu-head.l3-nav-mobile-closed').classList.toggle('not-active');
+  document.querySelector('.l3-nav-mobile-menu-head.l3-nav-mobile-open').classList.toggle('not-active');
+}
+
 /* ------------------------------ Global Functions ----------------------------------- */
 
-/**
- * A utility function to create divs with specified ID and classes
- * @param {string} id The id to associate with the created div element
- * @param {[string]} classes an array of strings indicating the classes to add to the div element
- * @returns {HTMLDivElement}
- */
-function createDiv(id, classes) {
```

Comment:
Deprecating this function in favor of the `div()` implementation provided by @synox .  Thanks!